src/main/features/core/tray.js

```diff
@@ -6,6 +6,24 @@ import { showDesktopSettings } from './desktopSettings';
 let appIcon = null;
 const mainWindow = WindowManager.getAll('main')[0];
 
+const trayNormalPath = path.resolve(`${__dirname}/../../../assets/img/`);
+const trayPausedPath = path.resolve(`${__dirname}/../../../assets/img/paused/`);
+const trayPlayingPath = path.resolve(`${__dirname}/../../../assets/img/playing/`);
+
+const changeTrayIcon = (iconFolderPath) => {
```

Comment:
well, didnt thought abou this.
note to me: late night working isnt really a thing for me 😆